core/livepeernode.go

```diff
@@ -67,7 +67,7 @@ type LivepeerNode struct {
 	TranscoderManager *RemoteTranscoderManager
 	PriceInfo         *big.Rat
 	Balances          *Balances
-
+	mu                sync.RWMutex
```

Comment:
Since we have the `SetBasePrice()` and `GetBasePrice()` methods now, can we unexport `PriceInfo` and group it with the rest of the unexported fields i.e. `serviceURI`?

It could also be helpful to add a comment noting that `mu` protects config fields

Reply:
fixed in https://github.com/livepeer/go-livepeer/pull/993/commits/54863dce0b699a81e16eb27e09bf64b6b28e79a8

Reply:
fixed in https://github.com/livepeer/go-livepeer/pull/993/commits/54863dce0b699a81e16eb27e09bf64b6b28e79a8